Modules/Sources/WordPressTesting/Empty.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+extension Bundle {
+    public static var test: Bundle { Bundle.module }
```

Comment:
`test` seems too generic. What do you think of naming it `wordPressTesting`?

Reply:
I didn't rename it, so I'd suggest sticking with this name for now. I think it's OK because you are only ever going to import it in a subset of files in tests targets. It doesn't affect anything else.

<img width="1631" alt="Screenshot 2024-12-12 at 6 03 13 PM" src="https://github.com/user-attachments/assets/234abe6a-dfc3-4461-a540-0fe3e82869f1" />